src/components/a-checkbox/_preview.html

```diff
@@ -16,13 +16,13 @@
   name="test-name2"
   disabled
 >
-  hello
+  hello hello <a href="#" target="_blank">hello hello hello hello hello hello hello hello</a> hello hello hello hello <a href="#" target="_blank">hello hello hello hello </a> hello hello hello hello.
```

Comment:
please don't commit these extensive examples

Reply:
I think is ok extensive example, but maybe use lorem ipsum

Reply:
Yeah, though in this case the checkbox nor radio are considered being multiline/multicolumn elements

Reply:
but "authors" happens my friend :). So lets support the "mistaken" multi line